### DISCUSSION TITLE
Node.js 18 Support

### DISCUSSION THREAD

**Discussion by ghost** [nextauthjs/next-auth#5329] [Help] [answered]
Less than a year away from EOL of Node.js 16.
Node.js 16 has 7 months early EOL due to OpenSSL vulnerability issue.
https://nodejs.org/ru/blog/announcements/nodejs16-eol/

Currently, When I try to npm install, I get an error "Unsupported engine" and I can't install it.
I would like to know when this nice library will support Node.js 18.

I know that this problem can be solved by simply adding the 18 version of the engine to the package.json.
If there's a reason we can't support Node.js 18 other than it's not tested, I'd like to know it
https://github.com/nextauthjs/next-auth/blob/main/package.json#L43

**Comment by m11m** [comment]
any updates on this? I would really like to use node 18 due to performance reasons with another package, but next-auth is blocking.

**Reply by mrr11k** [reply]
There was a fix merged two days ago, so I hope a new release will follow soon. :) 

https://github.com/nextauthjs/next-auth/commit/30ad639d163147841a096211e1db2c9f10c34cb7

**Comment by ghost** [comment, marked as answer]
Now supports Node.js 18, thanks contibuter.
https://github.com/nextauthjs/next-auth/pull/5656
